server.ts

```diff
@@ -36,6 +37,7 @@ async function createServer() {
   app.use("/auth", authController());
   app.use("/user", userController());
   app.use("/review", reviewController());
+  app.use("/upload", uploadController());
```

Comment:
Its called blob so /blob
```suggestion
  app.use("/blob", blobController());
```